TCMSApp/karma.conf.js

```diff
@@ -54,6 +54,10 @@ module.exports = function(config) {
 
         // Continuous Integration mode
         // if true, Karma captures browsers, runs the tests and exits
-        singleRun: false
+        singleRun: false,
+
+        ngHtml2JsPreprocessor: {
+            moduleName: 'templates'
+        },
```

Comment:
I see that now you have no literal with the template to inject into the test. 
Nice!
Can you please describe the solution in few details? :)